packages/frontend/src/common/components/Button.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+interface Params {
+  title: string
+}
+
+const Button = ({ title }: Params) => {
+  return (
+    <button className="btn bg-accent h-12 min-w-[192px] text-white normal-case hover:bg-accentHover">
+      {title}
+    </button>
+  )
+}
+
+export default Button
```

Comment:
No default export. Lets use named export. Also this component is declared as function expression but other are defined as function declaration. We should aligned it. I lean towards function declaration.